engine/src/main/java/org/terasology/audio/openAL/BaseSoundSource.java

```diff
@@ -272,11 +272,4 @@ public void dispose() {
             AL10.alDeleteSources(sourceId);
         }
     }
-
-    @Override
-    public int hashCode() {
```

Comment:
The equals method is not implemented. I assume it makes most sense to go with defaults.